vkcs/internal/services/compute/v2/flavors/results.go

```diff
@@ -0,0 +1,53 @@
+package flavors
```

Comment:
The common way in `gophercloud` to do this would be to specify `ExtraSpecsExt`, add a method `ExtractFlavorsInto` into `iflavors` package, if there is no such in `flavors`, that would look like
```go
type flavorExtended struct {
  flavors.Flavor
  iflavors.ExtraSpecsExt
}

var allFlavors []flavorExtended
err := iflavors.ExtractFlavorsInto(allPages, &allFlavors)
```

Reply:
Check SDNExt, for example, and ports data source.

Reply:
Flavors package doesn't have method like ports.ExtractPortsInto, it only has `ExtractFlavors(r pagination.Page)`. So the way I choose is not too difficult

Reply:
It's not about difficulty, it's about sticking to the common way. You can add similar function into `iflavors` package.

Reply:
done